### ISSUE TITLE
Falco supported events

### ISSUE THREAD

**Issue by leogr** [falcosecurity/falco#2213]
**What to document**

After discussing with @Andreagit97 about recent changes and minor breaking changes (eg,e.g.-A` will not be applied anymore when reading from `.scap` files), we decided to create a table, so it was clear what will be the expected outcome for Falco 0.33.0. 

Ref #2201 

The documentation (and maybe other places like the help usage of the command line) may need to be updated accordingly.

/milestone 0.33.0

cc @falcosecurity/falco-maintainers @falcosecurity/libs-maintainers

## Falco supported events

| Category | `evt.type` | Default (without -A) | With `-A` | `.scap` file |
| -------- | -------- | -------- | -------- | -------- | 
| `EC_PLUGIN` | `plugin`     | Yes | Yes | Yes |
| `EC_SYSCALL` | various     | Only syscalls in *simple_set*  | All | All, including `EF_OLD_VERSION` | 
| `EC_TRACEPOINT` | `procexit`, `switch`, `signaldeliver`, `page_fault`    | `procexit` | All |  All, including `EF_OLD_VERSION` |
| `EC_INTERNAL` | `container`, `useradded`, `userdeleted`, `groupadded`, `groupdeleted` and other internal events    | Only `container`, `useradded`, `userdeleted`, `groupadded`, `groupdeleted` | All | All, including `EF_OLD_VERSION` |

Notes:
 - `EF_OLD_VERSION` are never generated when live mode, but they may be present in `.scap` files
 - Since the new Falco version won't apply any userspace pre-filtering, `-A` is implicit when reading from `.scap`
 - `EC_PLUGIN` is not yet present in `libs`, but our plan is to insert it soon

**Comment by Andreagit97** [comment]
> EC_PLUGIN is not yet present in libs, but our plan is to insert it soon

Added here  https://github.com/falcosecurity/libs/pull/622

**Comment by jasondellaluce** [comment]
@Andreagit97 @leogr I think this is solved for the scope of Falco 0.33.0, but it may be worth adding it on the Falco website (or the some libs internal document) instead of closing this. What do you think?

**Comment by leogr** [comment]
> it may be worth adding it on the Falco website 

Definitively yes :+1:

**Comment by Andreagit97** [comment]
makes sense but before doing that we have to update it, since the actual logic is not exactly as described here, but for sure we can update this issue until the design is ended and after that, we can move it to the documentation :)

**Comment by jasondellaluce** [comment]
For clarity, I'm gonna move this issue to falcosecurity/falco-website.
